distributed-databases/cassandra/vpc.tf

```diff
@@ -0,0 +1,42 @@
+#---------------------------------------------------------------
+# VPC and Subnets
+#---------------------------------------------------------------
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 3.0"
+
+  name = local.name
+  cidr = local.vpc_cidr
+
+  azs              = local.azs
+  public_subnets   = [for k, v in local.azs : cidrsubnet(local.vpc_cidr, 8, k)]
+  private_subnets  = [for k, v in local.azs : cidrsubnet(local.vpc_cidr, 8, k + 10)]
```

Comment:
Lets' use hardcoded private and public subnets. You can use reference from other blueprints.